src/dragonball/src/kvm_context.rs

```diff
@@ -91,6 +92,18 @@ mod x86_64 {
     use std::collections::HashSet;
 
     impl KvmContext {
+        /// Create a virtual machine object witch type.
```

Comment:
s/witch/which

Reply:
actually here should be `with` 😂
And I have fixed this.